docs/developers/cloud-service.md

```diff
@@ -0,0 +1,66 @@
+# Cloud service
+
+In this demo, we will add an RDS database from Alibaba Cloud to our application.
+
+## What is cloud service
+
+Cloud service refers to the managed cloud resources. For example, you can buy a PostgreSql database from a cloud vendor instead of setting up your own. Cloud service are normally outside of your Kubernetes cluster, but logically they are still part of your application. KubeVela provides application centric view of your applications. It will treat every service the same.
+
+## Crossplane
+
+Crossplane is an open source Kubernetes add-on that extends any cluster with the ability to provision and manage cloud infrastructure, services, and applications using kubectl, GitOps, or any tool that works with the Kubernetes API. The benefit of using Crossplane is it will provide centralized control plane disregard where your cluster is.
+
+KubeVela will delegate the lifecycle management of cloud service to Crossplane.
+
+## Install Crossplane (This demo uses crossplane version 0.13)
+
+You will need a Kubernetes cluster ver> 1.16 (Minikube and Kind clusters are fine).
+Also you will need to have KubeVela installed on the cluster.
+To provision a cloud resource, you need to have the Access Key and Secret to your Alibaba cloud account.
+
+* Create crossplane namespace: `kubectl create ns crossplane-system`
+* Install crossplane helm chart: `helm install crossplane  charts/crossplane/ --namespace crossplane-system`
```

Comment:
This exists under examples/kubecondemo/ . Maybe Ask users to `cd $project_dir/examples/kubecondemo/ ` at the beginning?

Reply:
I'm not sure which folder will stay alive longer.